### PR TITLE
Update static-pod.md

### PR DESCRIPTION
**Pull request by chris-short**
Fixes Issue #31394

### CODE REVIEW

File: content/en/docs/tasks/configure-pod-container/static-pod.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ The `spec` of a static Pod cannot refer to other API objects
 
 {{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
 
-This page assumes you're using {{< glossary_tooltip term_id="docker" >}} to run Pods,
+This page assumes you're using {{< glossary_tooltip term_id="containerd" >}} to run Pods,
```
Comment (sftim) [inline content/en/docs/tasks/configure-pod-container/static-pod.md:45]: Is there any containerd assumption in the revised page?

File: content/en/docs/tasks/configure-pod-container/static-pod.md [hunk 2]
```diff
@@ -156,7 +156,7 @@ already be running.
 You can view running containers (including static Pods) by running (on the node):
 ```shell
 # Run this command on the node where the kubelet is running
-docker ps
+crictl ps
```
Comment (sftim) [inline content/en/docs/tasks/configure-pod-container/static-pod.md:159]: Does the switch to `crictl` change the output format? If so, we should revise that too.

**Review by avinashupadhya99 — APPROVED** [review]

File: content/en/docs/tasks/configure-pod-container/static-pod.md [hunk 3]
```diff
@@ -230,11 +229,11 @@ The running kubelet periodically scans the configured directory (`/etc/kubelet.d
 #
 mv /etc/kubelet.d/static-web.yaml /tmp
 sleep 20
-docker ps
+crictl ps
 # You see that no nginx container is running
 mv /tmp/static-web.yaml  /etc/kubelet.d/
 sleep 20
-docker ps
+crictl ps
```
Comment (sftim) [inline content/en/docs/tasks/configure-pod-container/static-pod.md:236]: Does the switch to `crictl` change the output format? If so, we should revise that too.

File: content/en/docs/tasks/configure-pod-container/static-pod.md (hunk 1 above)
Comment (chris-short) [reply content/en/docs/tasks/configure-pod-container/static-pod.md:45]: This is a mistake on my part. Talking and typing is never good.

File: content/en/docs/tasks/configure-pod-container/static-pod.md (hunk 2 above)
Comment (chris-short) [reply content/en/docs/tasks/configure-pod-container/static-pod.md:159]: The image name is excessively long. Do we have a convention to handle that? Example:  
\`docker.io/library/nginx@sha256:0d17b565c37bcbd895e9d92315a05c1c3c9a29f762b011a10c54a66cd53c9b31 \`

File: content/en/docs/tasks/configure-pod-container/static-pod.md (hunk 3 above)
Comment (chris-short) [reply content/en/docs/tasks/configure-pod-container/static-pod.md:238]: Resolved.

**Comment by shannonxtreme** [comment]
Page preview: https://deploy-preview-31410--kubernetes-io-main-staging.netlify.app/docs/tasks/configure-pod-container/static-pod/#behavior-of-static-pods

File: content/en/docs/tasks/configure-pod-container/static-pod.md [hunk 4]
```diff
@@ -156,24 +156,26 @@ already be running.
...
 ```
 
 The output might be something like:
 
 ```
-CONTAINER ID IMAGE         COMMAND  CREATED        STATUS         PORTS     NAMES
-f6d05272b57e nginx:latest  "nginx"  8 minutes ago  Up 8 minutes             k8s_web.6f802af4_static-web-fk-node1_default_67e24ed9466ba55986d120c867395f3c_378e5f3c
+CONTAINER           IMAGE                                CREATED             STATE               NAME                        ATTEMPT             POD ID
```
Comment (shannonxtreme) [inline content/en/docs/tasks/configure-pod-container/static-pod.md:165]: For these outputs, can you please compare to the rendered page at https://deploy-preview-31410--kubernetes-io-main-staging.netlify.app/docs/tasks/configure-pod-container/static-pod/#behavior-of-static-pods and reduce the column spacing? Thanks!

Comment (sftim) [reply content/en/docs/tasks/configure-pod-container/static-pod.md:165]: There might be a bug of sorts here (possibly already filed, I haven't checked). The sample output isn't using the full width of the parent element, and that's not helpful for wide items like this one.

As a workaround, try appending `console` to line 164 @chris-short (I haven't tested this). The idea is to trigger the syntax highlighter.

**Review by sftim — COMMENTED** [review]
/lgtm

**Comment by shannonxtreme** [comment]
LGTM (and also lol at commit message @chris-short )

**Comment by sftim** [comment]
/approve

Also taking LGTM from https://github.com/kubernetes/website/pull/31410#issuecomment-1022307327

**Pull request merged**